Solutions/moritz_w/Lesson_8/dictionaries.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+
+def word_frequencies(words):
+    ret = {}
+
+    for word in words:
+        lower_word = word.lower()
+        if lower_word not in ret:
+            ret[lower_word] = 1
+        else:
+            ret[lower_word] += 1
+    return ret
+
+
+def most_occurring_words(words_freq):
+    freq = 0
+    most_frequent_words = []
+
+    for word, amount in words_freq.items():
```

Comment:
A little simpler way to do it is to just find max value of frequency and iterate over words_freq and add to a list words that have that max frequency.